envs/example/ci-full-rhel/group_vars/all.yml

```diff
@@ -9,9 +9,12 @@ state_path_base: /opt/stack/data
 
 validate_certs: false
 
-redhat:
-  user: XXXXXXX
-  pass: XXXXXXX
+rhn_subscription:
+  username: XXXXXXX
+  password: XXXXXXX
+  pool_names: []
+  repos:
+    enable: []
```

Comment:
I believe having this empty array here will blank it out.  Dicts will merge but arrays will get clobbered.